DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: staged.dependencies
 Type: Package
 Title: Install R packages from Particular Git Branches
-Version: 0.2.4
+Version: 0.2.5
```

Comment:
actually should we set this to 0.2.4.9001 so that only when we "release" do we get 0.2.5?

Reply:
Ah I think that was in preparation for next release (there wasn't a version bump action etc. here)